test/clj/racehub/util_test.clj

```diff
@@ -90,8 +99,9 @@
       "large ints are printed in a sexy way."))
 
 (deftest currency-amt-test
-  (is (= "0.00" (currency-amt nil))
-      "Nil handling works.")
+  (s/without-fn-validation
```

Comment:
I originally didn't want maybe to leak into format-currency, but I kept it in, so I'll remove this.